WRN-1542: Sampler: resolve the warning related to babel (#14)
enactjs/storybook-utils: CHANGELOG.md

## unreleased

> * Remove Babel plugins because these were already included in `babel-preset-env` v7.15.0.

Reviewer: ```suggestion
* Removed some Babel plugins which were already included in the recent `@babe/preset-env`.
```

> * Adds `private-methods`, and `private-property-in-object` Babel plugins with {loose: true} option.

Reviewer: Please write the log in past tense.

```suggestion
* Added `private-methods` and `private-property-in-object` Babel plugins to remove warning
```
